HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -143,6 +149,54 @@ public async Task<IActionResult> TenantOrLeaseholdPropertyRepairCancel([FromQuer
             }
         }
 
+        [HttpPost]
+        [Route(nameof(TenantOrLeaseholdPropertyRepairChangeAppointmentSlot))]
+        public async Task<IActionResult> TenantOrLeaseholdPropertyRepairChangeAppointmentSlot([FromQuery] string postcode, [FromQuery] string repairId, [FromBody] RepairAvailability repairAvailability)
```

Comment:
The last parameter, `repairAvailability` is of type `RepairAvailability`, which is a domain model and shouldn't been exposed the endpoints.

At least a couple of options :
- create a class which encapsulates all the parameters required for this request, i.e. `TenantOrLeaseholdPropertyRepairChangeAppointmentSlotRequest`
- replace `repairAvailability` with 2 parameters for start and end dates, AppointmentsController in this [repo](https://github.com/Newarkandsherwood/housing-repairs-online-api/blob/main/HousingRepairsOnlineApi/Controllers/AppointmentsController.cs#L28) and [SchedulingAPI](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/blob/main/HousingRepairsSchedulingApi/Controllers/AppointmentsController.cs#L48) does this